src/iconsManifest/supportedExtensions.ts

```diff
@@ -1674,12 +1674,15 @@ export const extensions: IFileCollection = {
     {
       icon: 'gatsby',
       extensions: [],
-      filenamesGlob: [
-        'gatsby-config',
-        'gatsby-node',
-        'gatsby-browser',
-        'gatsby-ssr',
-      ],
+      filenamesGlob: ['gatsby-browser'],
+      extensionsGlob: ['js', 'ts', 'tsx'],
+      filename: true,
+      format: FileFormat.svg,
+    },
+    {
+      icon: 'gatsby',
+      extensions: [],
+      filenamesGlob: ['gatsby-config', 'gatsby-node', 'gatsby-ssr'],
```

Comment:
```suggestion
      filenamesGlob: ['gatsby-browser', 'gatsby-ssr'],
      extensionsGlob: ['js', 'ts', 'tsx'],
      filename: true,
      format: FileFormat.svg,
    },
    {
      icon: 'gatsby',
      extensions: [],
      filenamesGlob: ['gatsby-config', 'gatsby-node'],
```

Would be like this :)

Reply:
Do you mean gatsby-ssr must support tsx too?

Reply:
Done

Reply:
I saw it now in the referred PR

Reply:
Yeah, correct!